src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,57 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client Errors
+
+The SDK automatically captures HTTP Client errors and sends them to [sentry.io](https://sentry.io).
+
+The HTTP Client integration should have 3 configuration options:
+
+- `captureFailedRequests` defaults to `false` due to PII reasons.
+  - The SDK will only capture HTTP Client errors if it is enabled.
+- `failedRequestStatusCodes` defaults to `500 - 599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
+  - The SDK will only capture HTTP Client errors if the HTTP Response status code is within the defined ranges in `failedRequestStatusCodes`.
+  - If the language has a `Range` type, it should be used instead of `HttpStatusCodeRange`.
+- `failedRequestTargets` defaults to (`.*`), this configuration option accepts a `List` of `String` that may be Regular expressions as well, similar to <Link to="/sdk/performance/#tracepropagationtargets">tracePropagationTargets</Link>.
+  - The SDK will only capture HTTP Client errors if the HTTP Request URL is a match for any of the `failedRequestsTargets`.
+
+The HTTP Client integration should capture error events with the following properties:
+
+The Request interface, see the <Link to="/sdk/event-payloads/request/">Spec</Link> for details.
+
+The Response context (Not yet under the Spec):
+
+```json
+{
+  "contexts": {
+    "response": {
+      "type": "response",
+      "cookies": "PHPSESSID=298zf09hf012fh2; csrftoken=u32t4o3tb3gg43; _gat=1;",
+      "headers": {
+        "content-type": "text/html"
+        /// ...
+      },
+      "status_code": 500,
+      "body_size": 1000, // in bytes
+    }
+  }
+}
+```
+
+The Exception Interface, see the <Link to="/sdk/event-payloads/exception/">Spec</Link> for details.
+
+If the HTTP Client integration does not throw an exception for unsuccessful requests, you can create a synthetic exception following this Spec:
+
+- Set the <Link to="/sdk/event-payloads/exception/#exception-mechanism">Exception Mechanism</Link> with a proper `type` such as `SentryOkHttpInterceptor`.
+- Set the <Link to="/sdk/event-payloads/stacktrace/">Stack Trace Interface</Link> with `snapshot=true`.
+  - The Exception message can be `Event was captured because the request status code was $code`.
```

Comment:
`l`: Again only one list item looks a bit weird. I would also change the `can` to a `should`
```suggestion
- Set the <Link to="/sdk/event-payloads/stacktrace/">Stack Trace Interface</Link> with `snapshot=true`. The Exception message should be `Event was captured because the request status code was $code`.
```